bytestring_splitter/__init__.py

```diff
@@ -348,13 +348,18 @@ def repeat(self, splittable, as_set=False):
 class BytestringKwargifier(BytestringSplitter):
     processed_objects_container = dict
     partial_receiver = PartiallyKwargifiedBytes
+    __splitterbaseclass = BytestringSplitter
```

Comment:
What is happening here? :monocle_face:

Reply:
Kwargifier was returning hardcoded instances of BytestringSplitter which we need to be VersionedBytestringSplitter if it's a VersionedKwargifier.

Maybe there's some better style way to do that?  I actually don't think it needs to be a method as it is and could just directly refer to `self. __splitterbaseclass`

Reply:
...why doesn't `super` serve here?

Reply:
I tried super() and it didn’t work for some reason but I may have been mistaken as to why it didn’t work.  I’ll try it again.

Reply:
super doesn't work here.

Reply:
I made a commit showing `super()` implementation with all the tests passing (2bb8cac43f07d8fbb53ab913c28b3182c1d5d9f9) and opened a PR into your fork (https://github.com/vepkenez/bytestringSplitter/pull/1).

Am I missing something?